src/densities.jl

```diff
@@ -26,7 +32,7 @@ function compute_partial_density(basis, kpt, ψk, occupation)
     end
 
     # FFT and return
-    r_to_G(basis, ρk_real)
+    r_to_G!(ρ, basis, complex.(ρk_real))
```

Comment:
You don't need a `complex.` here just `complex` does it (either way a copy).

Reply:
good catch